action.yml

```diff
@@ -0,0 +1,6 @@
+name: "Run OpenPedCan Analysis"
+description: "Run analysis workflow"
+runs:
+  using: "docker"
+  image: "Dockerfile"
+  entrypoint: "scripts/run-analysis.sh"
```

Comment:
Uses the script at `scripts/run-analysis.sh` as an [entrypoint for the action](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsentrypoint=) without modifying the dockerfile. Good find on this flag @logstar!